staging/src/k8s.io/client-go/tools/remotecommand/remotecommand.go

```diff
@@ -43,11 +46,17 @@ type StreamOptions struct {
 
 // Executor is an interface for transporting shell-style streams.
 type Executor interface {
-	// Stream initiates the transport of the standard shell streams. It will transport any
-	// non-nil stream to a remote system, and return an error if a problem occurs. If tty
-	// is set, the stderr stream is not used (raw TTY manages stdout and stderr over the
-	// stdout stream).
+	// Deprecated: use StreamWithContext instead to avoid possible resource leaks.
+	// See https://github.com/kubernetes/kubernetes/pull/103177 for details.
 	Stream(options StreamOptions) error
```

Comment:
Is it worth keeping this method around instead of requiring external callers to update?

If we do want to keep it, should we create an issue to remove the deprecated method in a release or two?

Reply:
@enj Yes, I think we need an issue to notice the users who relies on this.
I'll create one after this PR is merged.